similarity/src/test/java/edu/illinois/cs/cogcomp/mrcs/LlmComparatorTest.java

```diff
@@ -0,0 +1,176 @@
+			+ "lines to slash prices to fill vacant berths. It was built by STX Finland for Royal Caribbean International and left the shipyard "
+			+ "in Finland on Friday. Officials hadn't expected any problems in passing the Great Belt bridge, but traffic was stopped for about "
+			+ "15 minutes as a precaution when the ship approached, Danish navy spokesman Joergen Brand said. Aboard the Oasis of the Seas, "
+			+ "project manager Toivo Ilvonen of STX Finland confirmed that the ship had passed under the bridge without any incidents. \"Nothing "
+			+ "fell off,\" he said. The enormous ship features various \"neighborhoods\" -- parks, squares and arenas with special themes. One of "
+			+ "them will be a tropical environment, including palm trees and vines among the total 12,000 plants on board. They will be "
+			+ "planted after the ship arrives in Fort Lauderdale. In the stern, a 750-seat outdoor theater -- modeled on an ancient Greek "
+			+ "amphitheater -- doubles as a swimming pool by day and an ocean front theater by night. The pool has a diving tower with "
+			+ "spring boards and two 33-foot (10-meter) high-dive platforms. An indoor theater seats 1,300 guests. One of the \"neighborhoods,\" "
+			+ "named Central Park, features a square with boutiques, restaurants and bars, including a bar that moves up and down three decks, "
+			+ "allowing customers to get on and off at different levels. Once home, the $1.5 billion floating extravaganza will have more, if "
+			+ "less visible, obstacles to duck: a sagging U.S. economy, questions about the consumer appetite for luxury cruises and criticism "
+			+ "that such sailing behemoths are damaging to the environment and diminish the experience of traveling. It is due to make its U.S. "
+			+ "debut on Nov. 20 at its home port, Port Everglades in Florida.";
+	// private final String hyp = "Toivo Ilvonen is the president of STX Finland
+	// .";
+	private final String hyp = "weapons of mass destruction head towards Congo";
+	private final double expectedSimpleScoreNoStopwords = 0.1; // not using
+																// WNSim
+	private final double expectedSimpleScoreWithStopwords = 0.2; // not using
+																	// WNSim
+
+	private static boolean isSetup = false;
+
+	@BeforeClass
+	public static void setUpBeforeClass() throws Exception {
+		llm = new LlmStringComparator();
+	}
+
+	@After
+	public void tearDown() throws Exception {
+	}
+
+	@Test
+	public void testCompareStrings() {
+		double score = 0.0;
+		try {
+			score = llm.compareStrings_(text, hyp);
+		} catch (Exception e) {
+			e.printStackTrace();
+		}
+		System.out.println("score is: " + score);
+
+		assertTrue((Math.abs(score - this.expectedSimpleScoreWithStopwords)) > 0.0);
+	}
+
+	@Test
+	public void testParagramLlm() {
+		Properties props = new Properties();
+		props.setProperty(SimConfigurator.WORD_METRIC.key, EmbeddingConstant.wordnet);
```

Comment:
The name suggests this is NOT testing LLM with paragram, but with WordNet.  If this is true, please create a test for paragram as well.